docs/documentation_create_page.md

```diff
@@ -125,15 +126,18 @@ Please note, if you want to use Markdown inside an HTML block, it has to be surr
 
 ### Images, icons and logos
 
-The images which are displayed on the pages are stored in various directories according to their purpose. If you want to use a logo and placed `logo:` in the file header then this image should be stored in `source/images/supported_brands`. The background must be transparent.
+Having a logo with the integration, makes an integration quickly identifiable with the end-user.
+From the documentation side of things, no specific configuration is needed to enable the use of a logo,
+however, the logo must exist in our Brands repository.
+
+To add a logo and icon for your integration, open up a pull request at: the [Home Assistant Brands](https://github.com/home-assistant/brands).
+
+For other images, displayed on the pages, are stored in various directories according to their purpose:
```

Comment:
Other images, ..., are stored in various directories according to their purpose:

Reply:
Thanks!